app/Transformers/ScoreTransformer.php

```diff
@@ -102,6 +102,7 @@ public function transformSolo(MultiplayerScoreLink|ScoreModel|SoloScore $score)
         if ($score instanceof SoloScore) {
             $extraAttributes['ranked'] = $score->ranked;
             $extraAttributes['preserve'] = $score->preserve;
+            $extraAttributes['processed'] = $score->processHistory()->first() !== null;
```

Comment:
Should be `$score->processHistory !== null` instead.

Reply:
I changed to `->exists()` in https://github.com/ppy/osu-web/pull/11151/commits/e25f0333a1bb8cfce9691425fce4264f2c3c072c since it seems better but yeah I dunno. Can't seem to get it working without some method call at the end.

Reply:
oh, whoops, forgot to mention you have to add `processHistory` here for the accessor to work:
https://github.com/ppy/osu-web/blob/97f41e3cb1f1fc985e984454313a90e908da014a/app/Models/Solo/Score.php#L242

`processHistory()` returns the relation so any eager loads with be ignored, `processHistory` will use the existing value if already loaded.

Reply:
Yeah okay, that helped, thanks :+1: